cm2350/peripherals/flash.py

```diff
@@ -1325,9 +1363,12 @@ def _getArrayBlockOffset(self, offset):
     ##########################################
 
     def _flash_read(self, va, offset, size):
-        return self.data[offset:offset+size]
+        value = self.data[offset:offset+size]
+        #logger.debug("0x%x:  FLASH read  [%x:%r] (%s)", self.emu.getProgramCounter(), offset, size, value.hex())
+        return value
 
     def _flash_write(self, va, offset, bytez):
+        logger.debug("0x%x:  FLASH write [%x] = %s", self.emu.getProgramCounter(), va, bytez.hex())
```

Comment:
yes, in this case I commented them out because even though it shouldn't happen often I was still concerned about delays. I do like having the program counter there so perhaps a static method of getting it will be used in the future if these log statements are useful.